version/version.go

```diff
@@ -0,0 +1,45 @@
+package version
+
+import (
+	"github.com/arduino/arduino-cli/i18n"
+)
+
+var (
+	defaultVersionString = "0.0.0-git"
+	versionString        = ""
+	commit               = ""
+	status               = ""
+	date                 = ""
+	tr                   = i18n.Tr
+)
+
+// Info FIXMEDOC
+type Info struct {
+	Application   string `json:"Application"`
+	VersionString string `json:"VersionString"`
+	Commit        string `json:"Commit"`
+	Status        string `json:"Status"`
+	Date          string `json:"Date"`
+}
+
+// NewInfo FIXMEDOC
+func NewInfo(application string) *Info {
+	return &Info{
+		Application:   application,
+		VersionString: versionString,
+		Commit:        commit,
+		Status:        status,
+		Date:          date,
+	}
+}
+
+func (i *Info) String() string {
```

Comment:
you could leverage this when printing the logs

Reply:
Thanks for the suggestion. Had to leave i18n as it was since this function uses it.